Jenkinsfile

```diff
@@ -364,6 +364,12 @@ tar -xzf packaging/target/apache-hive-*-nightly-*-src.tar.gz
           stage('PostProcess') {
             try {
               sh """#!/bin/bash -e
+                FAILED_FILES=`find . -name "TEST*xml" -exec grep -l "<failure" {} \\; 2>/dev/null | head -n 10`
+                for a in \$FAILED_FILES
+                do
+                  RENAME_TMP=`echo \$a | sed s/TEST-//g`
+                  mv \${RENAME_TMP/.xml/-output.txt} \${RENAME_TMP/.xml/-output-save.txt}
```

Comment:
this is bad - you should fix issues wiuth the debug job and not make a crappy do-it-all from the main job...

Reply:
can you clarify what's bad, please? I'm open to revert the commit and fix

Reply:
the main job supposed to run the tests reliably ; and not necessarily help debug their issues

Reply:
clear proof that there are issues with the `TestMiniTezCliDriver`: http://ci.hive.apache.org/job/hive-flaky-check/736/

can be broken testpairs if it can't be reproduced separately; but such things like this is just a joke

you will wait hours for some crappy barely usable results - enhance the flaky-check debug job to collect more usable stuff if needed....